src/x509.cc

```diff
@@ -45,6 +45,85 @@ std::string parse_args(const Nan::FunctionCallbackInfo<v8::Value>& info) {
   return *String::Utf8Value(info[0]->ToString());
 }
 
+
+
+NAN_METHOD(verify) {
+  Nan::HandleScope scope;
+  OpenSSL_add_all_algorithms();
```

Comment:
Actually I'm not sure which would be better, Node.js core doesn't give the best solution for this. Now the node-x509 and the core module `crypto` are in parallel to use OpenSSL. Loading `crypto` at user-land would not be user-friend in some ways?

Calling `OpenSSL_add_all_algorithms()` without directives looks good to me as well now :)